src/configmypy/argparse_config.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, infer_types=True, overwrite_nested_config=False, **additional
         self.infer_types = infer_types
         self.overwrite_nested_config = overwrite_nested_config
     
-    def read_conf(self, config=None, **additional_config):
+    def read_conf(self, config=None, fuzzy=False, **additional_config):
```

Comment:
Should we change to `infer_type={False, 'fuzzy', 'strict'}`?

Reply:
That sounds good.